lib/cisco_node_utils/cmd_ref/feature.yaml

```diff
@@ -74,3 +74,10 @@ vni:
     set_value: 'feature vn-segment-vlan-based'
   N8k: *feature_mt_lite
   N9k: *feature_mt_lite
+
+vtp:
+  kind: boolean
+  get_command: "show running vtp"
```

Comment:
I would use `'show running | i ^feature' instead.

Or... we could use a template for this file in which case most of these don't need to specify a get_command.